.gitignore

```diff
@@ -11,3 +11,4 @@ doc
 rdoc
 pkg
 Gemfile.lock
+.idea
```

Comment:
This seems unrelated, could we remove it for now and/or move it to a separate PR if we need to do it? (I'm not sure what .idea files are for off the top of my head). Thanks!